src/app/pages/datasets/components/dataset-form/name-length-validation.ts

```diff
@@ -0,0 +1,34 @@
+import { UntypedFormControl, ValidatorFn } from '@angular/forms';
+import { DefaultValidationError } from 'app/enums/default-validation-error.enum';
+
+export function datasetNameTooLong(parentPath: string): ValidatorFn {
+  const maxLengthAllowed = 200;
```

Comment:
can we be dynamic here and don't hardcode the 200 value?

Reply:
This is the limit for dataset name. The validator is also very specific as there's logic in here that can't be applied to other generic fields with length limit validations. So, I don't think we need to make this generic. If in the future, the dataset limit changes, we can just edit this one value and it'll still work.